scripts/localization

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+
+xcrun extractLocStrings -s WMFLocalizedString -o Wikipedia/iOS\ Native\ Localizations/Base.lproj/ Wikipedia/Code/*
+xcrun extractLocStrings -s WMFLocalizedString -a -o Wikipedia/iOS\ Native\ Localizations/Base.lproj/ WMF\ Framework/*
+xcrun extractLocStrings -s WMFLocalizedString -a -o Wikipedia/iOS\ Native\ Localizations/Base.lproj/ InTheNewsNotification/*
+xcrun extractLocStrings -s WMFLocalizedString -a -o Wikipedia/iOS\ Native\ Localizations/Base.lproj/ TopReadWidget/*
+xcrun extractLocStrings -s WMFLocalizedString -a -o Wikipedia/iOS\ Native\ Localizations/Base.lproj/ ContinueReadingWidget/*
+
```

Comment:
Runs the built in localization extract tool on all directories with source code. -s instructs it to look for `WMFLocalizedString` in addition to `NSLocalizedString`